src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1513,10 +1519,15 @@ void runtime::cpu::CPU_ExternalFunction::build()
 
     m_is_built = true;
 
-    if (m_release_function && !m_use_tbb)
+    if (m_release_function && !m_use_tbb && !(std::getenv("NGRAPH_DEX_DEBUG") != nullptr))
     {
         release_function();
     }
+    else if (std::getenv("NGRAPH_DEX_DEBUG") != nullptr)
+    {
+        string filename = file_util::path_join(s_debug_dir, m_function_name + "_debug.txt");
+        runtime::cpu::CPU_ExternalFunction::write_to_file(writer, s_debug_dir, filename);
+    }
```

Comment:
do we ever release a poor `m_release_function` if `NGRAPH_DEX_DEBUG` is set? :smile: